test/example_data.dart

```diff
@@ -620,6 +620,9 @@ UserTopicEvent userTopicEvent(
   );
 }
 
+MessageEvent messageEvent(Message message) =>
```

Comment:
commit-message nit:

    example_data [nfc]: Add messageEvent

Better:

    test [nfc]: Add eg.messageEvent

For examples of previous summary lines, try this command:
```
$ git log --format='%>(20)%an %cs %h %s' --grep 'eg\.|example' origin \
    | grep -P '\bexample|\beg\b'
```

Reply:
Thanks! I added a `-E` flag to the `git log` command to get this working.

Reply:
Ah yeah, thanks. I have this in my `~/.gitconfig`, which I recommend:
```
[grep]
        # "Basic" regexps are a relic.  Extended would be fine; Perl is better.
        patternType = perl
```

That's equivalent to a `-P` flag, which goes farther in the same direction as `-E`.